src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -253,11 +252,9 @@ public Measure define(Measure measure) {
      * Bind index rule to the stream
      *
      * @param stream     the subject of index rule binding
-     * @param beginAt    the start timestamp of this rule binding
-     * @param expireAt   the expiry timestamp of this rule binding
      * @param indexRules rules to be bounded
      */
-    public void defineIndexRules(Stream stream, ZonedDateTime beginAt, ZonedDateTime expireAt, IndexRule... indexRules) {
+    public void defineIndexRules(Stream stream, IndexRule... indexRules) {
```

Comment:
Time range has been removed from the API, please update the doc accordingly. Please recheck whethere there are any more APIs changes?

Reply:
The javadoc is correct for this method. This should have be removed.

Reply:
I mean this https://github.com/apache/skywalking-banyandb-java-client#indexrules

> Internally, an IndexRuleBinding is created automatically for users, which will be active between beginAt and expireAt. With this shorthand API, the indexRuleBinding object will be active from the time it is created to the far future (i.e. 2099-01-01 00:00:00.000 UTC).